episodes/pull-request.md

```diff
@@ -76,6 +76,12 @@ the "file" icon to the top right, which indicates to "display the rich diff".
 of a pyramid, one with a blue sky and yellow pyramid and the other with a yellow
 sky and lavender pyramid."}
 
+::::: callout
+
+If you use any sort of code that generates random numbers, you may end up with small changes that show up on the list of changed files. [See this example](https://github.com/MCMaurer/Rewrite-R-ecology-lesson/pull/3), where using the `ggplot2` function `geom_jitter()` leads to slightly different image files. You can fix this by setting a seed for the random number generator used in the episode, so that the same random numbers are generated each time the lesson is built.
```

Comment:
Clarify that the documents are R markdown documents and spell out `set.seed()` and where to include it. 

```suggestion
In R Markdown documents, If you use any sort of code that generates random numbers, you may end up with small changes that show up on the list of changed files. See this example where [using the `ggplot2` function `geom_jitter()` leads to slightly different image files](https://github.com/MCMaurer/Rewrite-R-ecology-lesson/pull/3). You can fix this by setting a seed for the random number generator (e.g. `set.seed(1)`) at the beginning of the episode, so that the same random numbers are generated each time the lesson is built.
```